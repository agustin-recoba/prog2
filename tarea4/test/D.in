insertarEnBinario (20,0)
recorridaPorNiveles
insertarEnBinario (5,0)
insertarEnBinario (2,0)
insertarEnBinario (7,0)
insertarEnBinario (8,0)
insertarEnBinario (9,0)
insertarEnBinario (10,0)
agregarAIterador 30
agregarAIterador 40
agregarAIterador 50
agregarAIterador 60
agregarAIterador 70
agregarAIterador 80
cantidadEnIterador
estaVaciaPila
apilar 20
estaVaciaPila
apilar 30
desapilar
cima
encolar 10
frente
desencolar
enAlguno 3 35 45 85
soloEnA 4 35 45 50 85
cantidadEnIterador
estaVaciaPila
estaLlenaPila
recorridaPorNiveles
estaVaciaColaBinarios
Fin
