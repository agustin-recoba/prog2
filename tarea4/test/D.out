1>Insertado.
2>20 
3>Insertado.
4>Insertado.
5>Insertado.
6>Insertado.
7>Insertado.
8>Insertado.
9>Agregando.
10>Agregando.
11>Agregando.
12>Agregando.
13>Agregando.
14>Agregando.
15>6.
16>p está vacia.
17>Apilado.
18>p NO está vacia.
19>Apilado.
20>Desapilado.
21>20.
22>Encolado.
23>10
24>Desencolado.
25>30 35 40 45 50 60 70 80 85 
26>30 40 60 70 80 
27>6.
28>p NO está vacia.
29>p NO está llena.
30>10 | 9 | 8 | 2 7 | 5 | 20 
31>q está vacia.
32>Fin.
